circle.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
      working_directory: /var/www
      docker:
-        - image: retreatguru/php71-apache
+        - image: retreatguru/connect-plugin:php81
```

Comment:
I'm assuming you made this in our dockerhub. It's a clone of a `php81-apache` image

Reply:
yes, there was a lot of cruft and unnecessary software installed in the previous docker image so I made one specifically for testing the connect plugin